src/org/usfirst/frc/team467/robot/Autonomous/AutoDrive.java

```diff
@@ -0,0 +1,20 @@
+package org.usfirst.frc.team467.robot.Autonomous;
+
+public interface AutoDrive {
+
+	void moveFeet(double distance);
+	void rotateByAngle(double rotationInDegrees);
+
+	void moveFeet(double leftDistance, double rightDistance);
```

Comment:
I think this could be protected scope. Does this really need to be part of the interface?